mapiproxy/servers/default/nspi/dcesrv_exchange_nsp.c

```diff
@@ -890,6 +974,12 @@ static void dcesrv_NspiGetSpecialTable(struct dcesrv_call_state *dce_call,
 		DCESRV_NSP_RETURN(r, MAPI_E_LOGON_FAILED, NULL);
 	}
 
+	nspi_address_creation_templates = r->in.dwFlags & NspiAddressCreationTemplates;
+	nspi_unicode_strings            = r->in.dwFlags & NspiUnicodeStrings;
```

Comment:
We don't use horizontal alignment in normal statements. (only for the declaration on the variables on top of each function).